{"name": "test_login01[6-7]", "status": "failed", "statusDetails": {"message": "AssertionError: assert (6 + 7) == 2", "trace": "self = <test_case.testfunc01.TestLogin object at 0x00000245FE778D30>, a = 6\nb = 7\n\n    @pytest.mark.parametrize('a,b', [(1, 1), (2, 4), (2, 6), (6, 7)])\n    def test_login01(self, a, b):\n        print('---test_login01---')\n>       assert a + b == 2\nE       assert (6 + 7) == 2\n\ntestfunc01.py:13: AssertionError"}, "parameters": [{"name": "a", "value": "6"}, {"name": "b", "value": "7"}], "start": 1622897861419, "stop": 1622897861420, "uuid": "bd4be90c-065d-4bfc-9d35-048f50b26570", "historyId": "1c708ac3b81f3c7aa15c4081da3b69d3", "testCaseId": "6476584284445ea88e6a99052c533646", "fullName": "test_case.testfunc01.TestLogin#test_login01", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "testfunc01"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-BFADA0J"}, {"name": "thread", "value": "10588-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.testfunc01"}]}